source/_integrations/tplink_omada.markdown

```diff
@@ -20,11 +20,12 @@ The integration provides basic configuration and status of Omada devices control
 
 {% include integrations/config_flow.md %}
 
-- Host: Enter the URL of the Omada management interface.
+- Host: Enter the URL, DNS Name or IP Address of the TP-Link Omada Controller. HTTPS will be used be default, if not specified.
```

Comment:
Unrelated change, this should not be in this PR.

Reply:
Fair point. I've reverted.